packages/moonstone/ToggleButton/ToggleButton.js

```diff
@@ -155,7 +155,6 @@ const ToggleButtonBase = kind({
 		disabled: false,
 		minWidth: true,
 		selected: false,
```

Comment:
Need to add a `defaultProp` here or else `getSize` will change to 'large'.  We could also update all the `setSize()` calls to be 'small' by default now.  However, just assigning a `defaultProps` should be fine.

Reply:
Interesting. That wasn't the case that I had seen on `ToggleButton`. It renders small, inheriting the default size from `Button`.

I was thinking that at a later time, we'd simply remove the `getSize` method entirely, but maybe it should just be deleted now. What do you think? (I'd prefer deleting it entirely, rather than change the code to better support it, as it's a temporary workaround to support the old `small` prop.)

Reply:
I didn't look deeply, it could be the default comes from how it's constructed/used.  I don't mind if we remove `getSize()` and just leave the deprecation code in.  For `ui`, we should remove the `getSize()` and use `defaultProps`, though it can wait until we remove the deprecations.